numpy/_core/_internal.py

```diff
@@ -905,11 +905,9 @@ def _ufunc_doc_signature_formatter(ufunc):
     elif ufunc.nout == 1:
         out_args = ', /, out=None'
     else:
-        out_args = '[, {positional}], / [, out={default}]'.format(
-            positional=', '.join(
-                'out{}'.format(i+1) for i in range(ufunc.nout)),
-            default=repr((None,)*ufunc.nout)
-        )
+        positional = ', '.join(
+                f'out{i+1}' for i in range(ufunc.nout))
+        out_args = f'[, {positional}], / [, out={repr((None,)*ufunc.nout)}]'
```

Comment:
nit: binding a `default` variable as well would be more readable IMO

Reply:
Yeah, good point.  I'll look into making it more readable.